internal/data/services.go

```diff
@@ -0,0 +1,22 @@
+package data
+
+import (
+	"database/sql"
+	"errors"
+)
+
+var (
+	ErrRecordNotFound = errors.New("record not found")
+)
+
+type Services struct {
```

Comment:
I'd probably break these out into separate packages, I think

Reply:
Do you mean separating the database and service code?

Reply:
Yep!